server/graphql/v2/mutation/CreateOrganizationMutation.js

```diff
@@ -0,0 +1,77 @@
+import { GraphQLBoolean, GraphQLNonNull } from 'graphql';
+import { pick } from 'lodash';
+
+import activities from '../../../constants/activities';
+import roles from '../../../constants/roles';
+import { isCollectiveSlugReserved } from '../../../lib/collectivelib';
+import models from '../../../models';
+import { Unauthorized } from '../../errors';
+import { OrganizationCreateInput } from '../input/OrganizationCreateInput';
+import { Organization } from '../object/Organization';
+
+const DEFAULT_ORGANIZATION_SETTINGS = {
+  features: { conversations: true },
+};
+
+async function createOrganization(_, args, req) {
+  const { remoteUser, loaders } = req;
+
+  if (!remoteUser) {
+    throw new Unauthorized('You need to be logged in to create an organization');
+  }
+
+  const organizationData = {
+    type: 'ORGANIZATION',
+    slug: args.organization.slug.toLowerCase(),
+    ...pick(args.organization, ['name', 'description', 'website']),
+    isActive: false,
+    CreatedByUserId: remoteUser.id,
+    settings: { ...DEFAULT_ORGANIZATION_SETTINGS, ...args.organization.settings },
+  };
+
+  if (isCollectiveSlugReserved(organizationData.slug)) {
+    throw new Error(`The slug '${organizationData.slug}' is not allowed.`);
+  }
+  const collectiveWithSlug = await models.Collective.findOne({ where: { slug: organizationData.slug } });
+  if (collectiveWithSlug) {
+    throw new Error(`The slug ${organizationData.slug} is already taken. Please use another slug for your collective.`);
+  }
+
+  const organization = await models.Collective.create(organizationData);
+
+  // Add authenticated user as an admin
+  await organization.addUserWithRole(remoteUser, roles.ADMIN, { CreatedByUserId: remoteUser.id });
+
+  // Will send an email to the authenticated user
+  // - tell them that their organization was successfully created
+  // - tell them that their organization is pending validation (which might be wrong if it was automatically approved)
+  const remoteUserCollective = await loaders.Collective.byId.load(remoteUser.CollectiveId);
+  models.Activity.create({
+    type: activities.COLLECTIVE_CREATED,
+    UserId: remoteUser.id,
+    data: {
+      collective: organization.info,
+      user: {
+        email: remoteUser.email,
+        collective: remoteUserCollective.info,
+      },
+    },
+  });
```

Comment:
For organizations, we don't send out the "collective created" email (see https://github.com/opencollective/opencollective-api/blob/edabab5bcf9dba6418eea2e88be58311a0300361/server/graphql/v1/mutations/collectives.js#L158)

Reply:
@sbinlondon, Here we aren't notifying the `host` but only the `admin`.

Reply:
I'm removing it as the user is redirected to the newly created org page with a banner at the top saying `Your org is created`